### PR TITLE
Update home page for 2025 series

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

**Comment by stefaniebutland** [comment]
Need to add HBCU logos, otherwise ready to publish after reviews I think.

File: index.qmd [hunk 1]
```diff
@@ -4,85 +4,81 @@ title: "Pathways to Open Science"
...
 
 ## Program Details
 
 This remote event series will convene Black environmental and marine researcher community members with the goal of developing a mindset for Open Data Science: the tools and practices enabling reproducible, transparent, and inclusive practices for data intensive science. If you're feeling bogged down by folders full of files with names like "analysis_final_v5.xlsx" and/or like empowering others, we welcome you to join us!
 
-**This 6-part remote event series runs on Thursdays, January 25 - February 29**. Event format will alternate weekly (all virtual):
+This is an ongoing Openscapes collaboration with Black in Marine Science ([BIMS](https://www.blackinmarinescience.org/)), Black Women in Ecology, Evolution, and Marine Science ([BWEEMS](https://www.bweems.org/)) and HBCU marine science professors. The program is led by Ileana Fenwick, with Alex Davis, PhD, and Aneese Williams, MSc.
```
Comment (jules32) [inline index.qmd:15]:
```suggestion
This is an ongoing [Openscapes](https://openscapes.org) collaboration with Black in Marine Science ([BIMS](https://www.blackinmarinescience.org/)), Black Women in Ecology, Evolution, and Marine Science ([BWEEMS](https://www.bweems.org/)) and HBCU marine science professors. The program is led by Ileana Fenwick, with Alex Davis, PhD, and Aneese Williams, MSc.
```

Comment (jules32) [inline index.qmd:15]:
```suggestion
This is an ongoing Openscapes collaboration with Black in Marine Science ([BIMS](https://www.blackinmarinescience.org/)), Black Women in Ecology, Evolution, and Marine Science ([BWEEMS](https://www.bweems.org/)) and marine science professors from 7 Historically Black Colleges and Universities (HBCUs). The program is led by Ileana Fenwick, with Alex Davis, PhD, and Aneese Williams, MSc.
```

File: index.qmd [hunk 2]
```diff
@@ -4,85 +4,81 @@ title: "Pathways to Open Science"
...
+[LaWanda Walker](https://www.linkedin.com/in/lawanda-walker-b673a71b2/) creates the promotional graphics and content for Pathways to Open Science and coordinates promotion with our collaborators. She is a graduate of Hampton University and current graduate student at the University of Chicago with an interest in nonprofit management and public history.
 
 [Julie Lowndes](https://www.openscapes.org/team/), PhD, founded and directs Openscapes and provides support for the Pathways to Open Science program.
```
Comment (jules32) [inline index.qmd:57]:
```suggestion
[Julie Lowndes](https://www.openscapes.org/team/), PhD, is Openscapes founder & core team member and provides support for the Pathways to Open Science program.
```

File: index.qmd [hunk 3]
```diff
@@ -4,85 +4,81 @@ title: "Pathways to Open Science"
...
 **BIMS**: [Website](https://blackinmarinescience.org) • [Instagram](https://instagram.com/blackinmarinescience?igshid=MWI4MTIyMDE=) • [Twitter](https://twitter.com/blackinmarsci) • [YouTube](https://www.youtube.com/BlackInMarineScience)
 
-**Openscapes**: [Website](https://openscapes.org) • [Mastodon](https://fosstodon.org/@openscapes) • [YouTube](https://www.youtube.com/@openscapes)
+**Openscapes**: [Website](https://openscapes.org) • [Bluesky](https://bsky.app/profile/openscapes.bsky.social) • [Mastodon](https://fosstodon.org/@openscapes) • [YouTube](https://www.youtube.com/@openscapes)
 
 ## Background Information
 
 To learn more about Openscapes approaches:
 
+-   [Shifting institutional culture to develop climate solutions with Open Science](https://doi.org/10.1002/ece3.11341) (Lowndes et al. 2024, Ecology and Evolution)
```
Comment (jules32) [inline index.qmd:81]: let's add White House blog post first! And Antoinette's post as well!

File: index.qmd [hunk 4]
```diff
@@ -92,7 +88,7 @@ To learn more about Openscapes approaches:
...
+The 2025 Pathways event series is funded by NOAA Fisheries, NASA Earthdata and Openscapes LLC. We are grateful to Pathways to Open Science was kicked off with 2023 support from the [Code for Science & Society Event Fund](https://eventfund.codeforscience.org/event-fund-grantees/) through the [Chan Zuckerberg Initiative](https://chanzuckerberg.com/). Our 2024 Series was funded in part by [Mozilla Foundation's Alumni Connection grant](https://foundation.mozilla.org/en/blog/mozilla-announces-first-ever-alumni-connection-grant-finalists/) that supported our collaboration with [PREreview](https://prereview.org/) and Dr. Antoinette Foster’s [expertise](https://openscapes.org/blog/2024-08-09-antoinette-foster-letter-make-the-world-better-place/) in values-guided career navigation and anti-racist scholarly review. Thank you!
```
Comment (jules32) [inline index.qmd:91]:
```suggestion
The 2025 Pathways event series is funded by NOAA Fisheries and Openscapes LLC. We are grateful to our previous funders in past years as well. This program was kicked off in 2023, when it was funded in part by the [Code for Science & Society Event Fund](https://eventfund.codeforscience.org/event-fund-grantees/) through the [Chan Zuckerberg Initiative](https://chanzuckerberg.com/). In 2024 it was funded in part by [Mozilla Foundation's Alumni Connection grant](https://foundation.mozilla.org/en/blog/mozilla-announces-first-ever-alumni-connection-grant-finalists/) that supported our collaboration with [PREreview](https://prereview.org/) and Dr. Antoinette Foster’s [expertise](https://openscapes.org/blog/2024-08-09-antoinette-foster-letter-make-the-world-better-place/) in values-guided career navigation and anti-racist scholarly review. Thank you!
```

**Review by jules32 — COMMENTED** [review]
Looks great Stef. I added a few edits and comments. Thanks!

File: index.qmd (hunk 1 above)
Comment (stefaniebutland) [reply index.qmd]: added in my local branch; couldn't auto-commit

**Comment by stefaniebutland** [comment]
closes #49 & #61

**Pull request merged**